doc/source/methods/Anchors.ipynb

```diff
@@ -49,6 +49,40 @@
     "As highlighted by the above example, an anchor explanation consists of *if-then rules*, called the anchors, which sufficiently guarantee the explanation locally and try to maximize the area for which the explanation holds. This means that as long as the anchor holds, the prediction should remain the same regardless of the values of the features not present in the anchor. Going back to the sentiment example: as long as *not good* is present, the sentiment is negative, regardless of the other words in the movie review."
```

Comment:
<ol><li>Should we have a "loose" definition of a "predicate" as well before using it in the definitions of precision/coverage?</li><li>With respect to the FAQ link, I think we discussed that the 2nd reason for the empty anchor is actually not possible ("The predicted class of the data point always changes regardless of the perturbations applied to it."). Would you be able to edit the FAQ entry to reflect that?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>